src/Context.ts

```diff
@@ -22,20 +22,19 @@ import { FunctionInfo } from './FunctionInfo';
 import { Request } from './http/Request';
 import { Response } from './http/Response';
 import EventEmitter = require('events');
-import LogLevel = rpc.RpcLog.Level;
 
 export function CreateContextAndInputs(
     info: FunctionInfo,
-    request: rpc.IInvocationRequest,
-    userLogCallback: UserLogCallback
+    request: RpcInvocationRequest,
+    userLogCallback: UserLogCallback,
+    doneEmitter: EventEmitter

```

Comment:
For the most part, this function seem to be copy-pasted from the old worker. The only difference I see here is that, before, `doneEmitter` was created by this function, whereas now it's passed to it. Curious what difference it makes and why this decision was made?